invenio_db/version.py

```diff
@@ -30,4 +30,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "1.0.0a8"
+__version__ = "1.0.0a9.dev20150000"
```

Comment:
Better `1216` than `0000`, although chances are we won't be upgrading YYYMMDD much before `a9` is out...